### PR TITLE
bg(tests): raise coverage

### PR DESCRIPTION
**Pull request by emukungu**
- raise coverage
- fix hound issues

### CODE REVIEW

File: container/__tests__/components/layout.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import { mount } from 'enzyme'
```
[the remark this answers is not in the document]
Comment (aldookware) [reply container/__tests__/components/layout.test.js]: fix!

File: container/__tests__/components/homesidebar.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const mockStore = configureStore();
+const store = mockStore(initialState);
+
+describe('Sidebar tests', () => {
+  let sidebarWrapper;
+  beforeAll(() => {
+    sidebarWrapper = mount(
+      <Provider store={ store }>
+        <HomeSidebar />
+      </Provider>)
```
[the remark this answers is not in the document]
Comment (aldookware) [reply container/__tests__/components/homesidebar.test.js]: hello!

**Comment by aldookware** [comment]
> The PR diff size of 14921 lines exceeds the maximum allowed for the inline comments feature.

kindly fix this.

File: container/__tests__/components/homesidebar.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (emukungu) [reply container/__tests__/components/homesidebar.test.js]: resolved

**Pull request merged**